application/handlers.go

```diff
@@ -65,6 +65,44 @@ var UploadHandler = func(res muxer.Response, req *http.Request, app *Application
 	res.ResponseWriter.Write(content)
 }
 
+var DeleteHandler = func(res muxer.Response, req *muxer.Request, app *Application) {
+	if !app.EnableDelete {
```

Comment:
We can remove this test by removing directly the route L104 when delete is disabled.